metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -394,7 +394,7 @@ public Task<SearchResult> search(
     }
 
     OperationContext opContext = OperationContext.asSession(systemOperationContext,
-                    RequestContext.builder().buildRestli(ACTION_SEARCH, entityName), authorizer, auth, true)
+                    RequestContext.builder().buildRestli(auth.getActor().toUrnStr(), getContext(),ACTION_SEARCH, entityName), authorizer, auth, true)
```

Comment:
Small nit, in a few cases there's no space after the comma, e.g. `getContext(),ACTION_SEARCH`

Reply:
linter let me down